### ISSUE TITLE
The assert code is lacking from the Jupyter-notebook I downloaded

### ISSUE THREAD

**Issue by baltazardydensborg** [isdsucph/isds2021#14]
Hi there, 

Because of an administrative issue, I was first added to the Absalon Course the other day. As I wanted to prepare during July and just thought the info was late, I figured that the Assignement 0 from last year which was linked in the course description was a good place to start, so I did most of that. 

Now I've just started on this years assignment 0, and I've noted that the "assert" codelines which were present in last years assignment and checked the answers were missing. There are still graphical lines for them, but the actual code isn't there. I've tried re-dowloading it but they're still missing.

Is this intentional, or do you know if there's an issue with the file?

Attached are screenshots of the assignment  from last year and this year. 
<img width="1074" alt="Screenshot 2021-07-20 at 13 36 46" src="https://user-images.githubusercontent.com/30194886/126318054-38528f20-85a8-4a6a-9fef-10adc5d5f7b4.png">
<img width="1105" alt="Screenshot 2021-07-20 at 13 41 11" src="https://user-images.githubusercontent.com/30194886/126318060-6263e64d-5d3b-43f2-9a8c-8350f6bbcb56.png">

**Comment by lassearpe** [comment]
Hey Baltazar

It seems intentional to me. 

I have this part in my notebook:
![image](https://user-images.githubusercontent.com/77959268/126327751-543d1793-8556-483f-ae67-9e7af72aff28.png)

Best,
Lasse

**Comment by baltazardydensborg** [comment]
Ah okay - I must have missed that from the preamble, because it's different on the 2020 version - thank you for the quick answer!

**Comment by lassearpe** [comment]
NP :)

**Comment by joachimkrasmussen** [comment]
Yes, that's exactly right! Also, notice that most of the questions this year are different from last year. However, they are _very_ similar in spirit, so if you already practiced working on the assignment from last year, it should be fairly straightforward to adapt your answers to the version from this year.

Best,
Joachim
